danuri-rest/src/main/kotlin/org/aing/danurirest/global/third_party/sms/SendSmsUsecase.kt

```
@@ -0,0 +1,42 @@
+                text = text,
+            )
+
+        try {
+            val messageService =
+                NurigoApp.initialize(
+                    apiKey = solApiKey,
+                    apiSecretKey = solApiSecretKey,
+                    domain = "https://api.solapi.com",
+                )
+            messageService.send(message)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)